### PR TITLE
Release 3.0.0-rc.5

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.0.0-rc.5 (2019-01-10)
+
+- Fixed flyout component resize handle to not have extra styles in Firefox. [#15](https://github.com/blackbaud/skyux-flyout/pull/15)
+- Fixed flyout component component to only listen to `mousemove` and `mouseup` events while being resized. [#16](https://github.com/blackbaud/skyux-flyout/pull/16)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: Is there a way to describe this bug fix without mentioning the internal workings? What was the overarching issue being solved?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Change detection/listeners were firing on every mouse move even when they shouldn't. I just copied what we have in `grids` changelog for this same change. Open to any other ideas. Maybe @blackbaud-johnly has some guidance when he looks at it?

Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Gotcha. I was thinking more along the lines of "performance enhancements". Either way is fine with me.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 3.0.0-rc.5 (2019-01-10)
+
+- Fixed flyout component resize handle to not have extra styles in Firefox. [#15](https://github.com/blackbaud/skyux-flyout/pull/15)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: "flyout component resize handle" is a mouthful, and I couldn't tell what was a verb and what was a noun the first time I read this.

I'd suggest changing this to "Fixed styles on flyout's resize handle in Firefox."

File: CHANGELOG.md (hunk 1 above)
Comment (blackbaud-johnly) [reply CHANGELOG.md]: I'd tighten this up to "Fixed flyout component to only listen to `mousemove` and `mouseup` events when resizing."

Steve, if we want to obfuscate the internal workings, I'd suggest "Updated flyout component to improve performance." I defer to you on which version is more useful, but if we go with the second version, I think we should also update the entry in the grids changelog.

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I went ahead with "Updated flyout component to improve performance." 

I'll make a note to update grids later today

**Comment by Blackbaud-TrevorBurch** [comment]
@blackbaud-johnly ready for you again

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# 3.0.0-rc.5 (2019-01-10)
+
+- Updated flyout component to improve performance. [#16](https://github.com/blackbaud/skyux-flyout/pull/16)
+- Fixed styles on flyout component's resize handle in Firefox. [#15](https://github.com/blackbaud/skyux-flyout/pull/15)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: > Fixed resize handle of flyout component to display properly (Firefox).

I found this pattern in our skyux2 changelog. What do you think?

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Never mind. I saw that John already approved. Disregard!

**Pull request merged**